CRM/Activityreport/Data.php

```diff
@@ -62,31 +58,26 @@ public static function get($offset = 0, $limit = 0, $multiValuesOffset = 0, $sta
   }
 
   /**
-   * Get datetime of the oldesc Activity or today date if there is no Activity
-   * found (YYYY-MM-DD HH:ii:ss format).
+   * Return an array containing API date filter conditions basing on specified
```

Comment:
no need to mention that this method return an array or null .. it is already known via @return array|NULL , you just need to describe what this method actually does